factory_client.py

```diff
@@ -70,6 +70,32 @@ def _set_apps_commit_hash(self):
         def __getitem__(self, item):
             return self.json[item]
 
+    class Release:
```

Comment:
You can do this with a lot less boiler plate code with something like
```
class Release(NamedTuple):
    lmp_version: int

    @classmethod
    def parse(cls, release_info) -> "Release":
       ...
       return cls(lmp_version)
  
```

Reply:
OK, I just usually prefer explicit API, and use "attribute" to explicitly specify public attributes. I think using of dynamic attributes in init class will lead even to lesser boiler plate code in this particular case :). Also, we may use `dataclass` decorator.
Anyway, it's not really important, I'll do it as you suggested.

Reply:
I updated the PR according to your suggestion.